x-pack/plugins/monitoring/public/components/no_data/blurbs/cloud_deployment.js

```diff
@@ -32,7 +32,7 @@ export const CloudDeployment = () => {
             defaultMessage="section for a deployment to configure monitoring. For more information visit "
           />
           <EuiLink
-            href="https://www.elastic.co/guide/en/cloud/current/ec-enable-monitoring.html"
+            href="https://www.elastic.co/guide/en/cloud/current/ec-enable-logging-and-monitoring.html"
```

Comment:
I think we should use our global statics here to maintain consistency, eg:
```suggestion
            href=`${docLinks.ELASTIC_WEBSITE_URL}guide/en/cloud/${docLinks.DOC_LINK_VERSION}/ec-enable-logging-and-monitoring.html`
```

Reply:
Love it! I'll update

Reply:
I can't seem to use `DOC_LINK_VERSION`  as it resolves to `master` instead of `current` which breaks the URL

Reply:
It will resolve to `current` once it's built, but you can do something like this to override master:
```javascript
const version = (docLinks.DOC_LINK_VERSION === 'master') ? 'current' : docLinks.DOC_LINK_VERSION;
```